test/broadcasts.js

```diff
@@ -4,11 +4,13 @@ const Hubspot = require('..')
 
 describe('broadcasts', function() {
   describe('get', function() {
-    before(nockHelper.mockEndpoint('/broadcast/v1/broadcasts', []))
+    before(nockHelper.mockOauthEndpoint('/broadcast/v1/broadcasts', []))
     after(nockHelper.resetNock)
 
     it('Should return details on a set of broadcast messages', function() {
-      const hubspot = new Hubspot({ apiKey: 'demo' })
+      const hubspot = new Hubspot({
+        accessToken: process.env.ACCESS_TOKEN || 'fake-token',
```

Comment:
The test passes since we've mocked out the responses from HubSpot. If you try to run the tests with NOCK_OFF set to true then the tests would fail with an auth error.

Reply:
Cool. I can't remember, do HubSpot Access Tokens expire? (if so, would be great to document how to update `ACCESS_TOKEN` in Travis).

Reply:
They do expire! But I haven't set this in travis since all the tests using `accessToken` are mocked out via nock so we don't actually hit Hubspot's api. I'm intending to add some documentation around running the test suite locally with a generated ACCESS_TOKEN and nock turned off.

Reply:
That makes sense. As discussed, would be worth expanding upon this in the "contributing" section of the doc.